client/src/main/java/org/apache/rocketmq/client/impl/consumer/DefaultLitePullConsumerImpl.java

```diff
@@ -1021,16 +1032,154 @@ public MessageQueue getMessageQueue() {
         }
     }
 
+    public void pullMessage(final MessageQueue messageQueue) {
```

Comment:
Is it possible to abstract a method for the purpose of code reuse rather than duplicating it?

https://github.com/apache/rocketmq/blob/a3228ad2736c0311592b12f994361b06cf4f72a9/client/src/main/java/org/apache/rocketmq/client/impl/consumer/DefaultLitePullConsumerImpl.java#L896-L1020

Reply:
Thanks for your reply, I don't have a good idea, my thinking is to add a parameter to judge sync or async, and then if... else handle differently